airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -476,7 +476,7 @@ def run_check_dbt_command(self, normalization_image: str, command: str, cwd: str
         """
         Run dbt subprocess while checking and counting for "ERROR", "FAIL" or "WARNING" printed in its outputs
         """
-        if normalization_image.startswith("airbyte/normalization-oracle"):
+        if normalization_image.startswith("airbyte/normalization-oracle") or normalization_image.startswith("airbyte/normalization-clickhouse"):
```

Comment:
I think two is fine to combine with the `or` clause, but if we need to add another it might be worth switching to a collection

```
if any([normalization_image.startswith(x) for x in ["airbyte/normalization-oracle", "airbyte/normalization-clickhouse"]]) 
```

Reply:
once these live as connectors code it won't be a problem 😉 (but also yes. this entire section is kind of iffy. It kind of looks like recent dbt versions removed the `--event-buffer-size` option entirely)